source/index.rst

```diff
@@ -46,6 +46,12 @@ Introduction and Getting Started
    the_need_for
    getting_started
 
+.. toctree::
+   :maxdepth: 1
+
+   using_invest
```

Comment:
Mentioned this somewhere else too, but I think `getting_started` and `using_invest` should probably be integrated.